src/ExpressionEvaluator/Core/Source/FunctionResolver/MetadataResolver.cs

```diff
@@ -184,7 +189,7 @@ private bool MatchesTypeName(TypeDefinition typeDef, string name)
         {
             if ((methodDef.Attributes & MethodAttributes.RTSpecialName) != 0)
             {
-                if (!_reader.StringComparer.Equals(methodDef.Name, ".ctor"))
+                if (!_reader.StringComparer.Equals(methodDef.Name, ".ctor", ignoreCase: false))
```

Comment:
I don't quite follow when we need to use ```_stringComparer``` vs ```_reader.StringComparer```.  If there's a principle, can we add a comment to the declaration of ```_stringComparer```?